### DISCUSSION TITLE
Raum Soll Temp not visible anymore

### DISCUSSION THREAD

**Discussion by pbleij** [mbuchber/ha_heliotherm#22] [Q&A] [answered]
Since some time I do not see the Raum Soll Thermostat anymore. I do see teh one for WW bereitung. I editted the config.py file but also saw the additions of HKR1 and HKR2. So, not sure what has changed in the configs that somehow the Raum HKR Soll temp cannot be found anymore. Is there a way that I can reload all config files or is there a way to overwrite existing files?

**Comment by mbuchber** [comment, marked as answer]
Fixed...

**Reply by pbleij** [reply]
Thank you very much.
Updated the integration and we're Happy (Bleij) again.

**Reply by pbleij** [reply]
Only learning for me on scripting is: what was changed?
